src/main/java/model/Book.java

```diff
@@ -26,4 +30,5 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    private boolean isDeleted;
```

Comment:
shouldn't you specify the column name like:
```suggestion
    @Column(name = "is_deleted")
    private boolean isDeleted;
```

or it works as it is?

Reply:
I think i should to do this)